components/cosmetic_filters/renderer/cosmetic_filters_js_handler.h

```diff
@@ -69,6 +72,10 @@ class CosmeticFiltersJSHandler {
   std::vector<std::string> exceptions_;
   GURL url_;
   std::unique_ptr<base::DictionaryValue> resources_dict_;
+
+  // Shows if the content_cosmetic.bundle.js has injected in the current frame.
```

Comment:
"True if `content_cosmetic.bundle.js` has been injected in the current frame."

Reply:
Fixed